test/automation/src/task.ts

```diff
@@ -50,7 +50,7 @@ export class Task {
 				await this.terminal.assertSingleTab({ color: expected[0].icon.color, icon: expected[0].icon.id || 'tools' });
 			}
 		}
-		await this.quickinput.closeQuickInput();
+		await setTimeout(() => this.quickinput.closeQuickInput(), 500);
```

Comment:
Timeouts like this are inherently flaky, is there some other condition we could wait on?

Reply:
I fixed the cause of the issue so this shouldn't be needed anymore at all